anti-analysis/anti-vm/vm-detection/check-for-windows-sandbox-via-genuine-state.yml

```diff
@@ -15,10 +17,19 @@ rule:
       - 773290480d5445f11d3dc1b800728966:0x140001140
   features:
     - and:
+      # static
       - basic block:
         - and:
           - api: SLIsGenuineLocal
       - basic block:
         - and:
           - api: UuidFromString
           - string: "55c92734-d682-4d71-983e-d6ec3f16059f"
+      # dynamic
+      - call:
+        - and:
+          - api: SLIsGenuineLocal
+      - call:
+        - and:
+          - api: UuidFromString
+          - string: "55c92734-d682-4d71-983e-d6ec3f16059f"
```

Comment:
this duplication is obviously unfortunate, but i understand why

Reply:
potentially there's just a lot of duplication for `basic block` -> `call`

could we support `basic block / call` syntax to simplify this?